### DISCUSSION TITLE
ethProviderModule.default.init is not a function using @web3-react/walletconnect-v2

### DISCUSSION THREAD

**Discussion by jpangelle** [WalletConnect/walletconnect-monorepo#2788] [Developer Support]
We integrated Wallet Connect V2 using [`@web3-react/walletconnect-v2`](https://github.com/Uniswap/web3-react/blob/main/packages/walletconnect-v2). However, when clicking the Wallet Connect button, a modal does not show. Nothing happens on the UI and the console logs this:

```js
TypeError: Qt.default.init is not a function
    at yi.createClient (vendor.edb88cef.js:383:30697)
    at yi.initialize (vendor.edb88cef.js:383:30550)
    at yi.init (vendor.edb88cef.js:383:27729)
    at Zr.initialize (vendor.edb88cef.js:383:54958)
    at Zr.init (vendor.edb88cef.js:383:50736)
    at xr.<anonymous> (vendor.edb88cef.js:383:60014)
    at Generator.next (<anonymous>)
    at oi (vendor.edb88cef.js:349:2184)
```

Upon digging into the logs, it roughly translates to `TypeError: ethProviderModule.default.init is not a function` when unbundled.

Also, this error only happens in production on a deployed site. It does not happen when running locally. Running out of ideas on troubleshooting this. 🙏

**Comment by glitch-txs** [comment]
Hi could you share a minimal reproducible example repo?

**Comment by torreyatcitty** [comment]
This looks like the exact same issue we ran into also: https://github.com/Uniswap/web3-react/issues/840

wallectconnect-v2 seems to work fine and show a QRCode to scan, but not when we deploy to an ipfs url.

**Comment by crypblizz8** [comment]
Same question to above:

Also can you share your web3-react / wc2 setup code here?

**Comment by kILL27** [comment]
Good luck with solving the problem!
